src/worker.c

```diff
@@ -119,7 +127,14 @@ static void worker_entry(void *arg) {
 
     /* Load the file and eval the file when the loop runs. */
     JSValue filename = JS_NewString(ctx, wd->path);
-    CHECK_EQ(JS_EnqueueJob(ctx, worker_eval, 1, (JSValue *) &filename), 0);
+    JSValue blob = JS_UNDEFINED;
+    if (wd->blob)
```

Comment:
Where is this set?

Reply:
In the constructor of the worker, when creating the `worker_data`